practice-two/src/index.html

```diff
@@ -1,187 +1,364 @@
+                </p>
+              </div>
+              <button class="btn project-btn btn-base">review</button>
+              <figure class="content__item-img">
+                <img
+                  src="assets/images/image-product-2.webp"
+                  alt="thump FTA key image"
+                  class="img-default img-border product-img"
+                />
+                <div class="project__view">
+                  <figure class="quantity-view-img">
+                    <img
+                      src="assets/images/view.svg"
+                      alt="quantity view project"
+                      class="img-default quantity-view-icon"
+                    />
+                  </figure>
+                  <p class="text-normal text-color-base">91312</p>
+                </div>
               </figure>
-              <div class="testimonials__contact">
-                <p class="text-color-primary testimonials__contact-name">Ophelia Holland</p>
-                <p class="text-color-base testimonials__contact-mail">@ophelia_holland</p>
+            </div>
+            <div class="project__item">
+              <div class="project__item-description">
+                <p class="project__item-title">Fta key</p>
+                <p class="text-normal project__item-author">
+                  Ivan Alexander, Steven Soto, Clifford Hammond
+                </p>
+              </div>
+              <button class="btn project-btn btn-base">review</button>
+              <figure class="content__item-img">
+                <img
+                  src="assets/images/image-product-3.webp"
+                  alt="thump FTA key image"
+                  class="img-default img-border product-img"
+                />
+                <div class="project__view">
+                  <div class="quantity-view-img">
+                    <img
+                      src="assets/images/view.svg"
+                      alt="quantity view project"
+                      class="img-default quantity-view-icon"
+                    />
+                  </div>
+                  <p class="text-normal text-color-base">91312</p>
+                </div>
+              </figure>
+            </div>
+            <div class="project__item">
+              <div class="project__item-description">
+                <p class="project__item-title">Fta key</p>
+                <p class="text-normal project__item-author">
+                  Ivan Alexander, Steven Soto, Clifford Hammond
+                </p>
               </div>
+              <button class="btn project-btn btn-base">review</button>
+              <figure class="content__item-img">
+                <img
+                  src="assets/images/image-product-4.webp"
+                  alt="thump FTA key image"
+                  class="img-default img-border product-img"
+                />
+                <div class="project__view">
+                  <div class="quantity-view-img">
```

Comment:
You can use background image of pseudo before for the text `p class="text-normal text-color-base">91312</p>`